MapboxCoreNavigation/MMEEventsManager.swift

```diff
@@ -49,8 +49,8 @@ struct EventDetails {
         simulation = routeController.locationManager is ReplayLocationManager || routeController.locationManager is SimulatedLocationManager ? true : false
         
         sessionIdentifier = session.identifier.uuidString
-        originalRequestIdentifier = nil
-        requestIdentifier = nil
+        originalRequestIdentifier = session.originalRoute.routeIdentifier
+        requestIdentifier = routeController.routeProgress.route.routeIdentifier
```

Comment:
Looks like we can either use the SessionState.routeIdentifiers here, or remove the properties from SessionState altogether.

Reply:
👍, removing the props `originalRequestIdentifier` and `requestIdentifier` completely.